package.json

```diff
@@ -39,14 +39,17 @@
         "@aws-cdk/aws-iam": "1.119.0",
         "@aws-cdk/aws-logs": "1.119.0",
         "@aws-cdk/aws-secretsmanager": "1.119.0",
-        "@aws-cdk/core": "1.119.0",
+        "@aws-cdk/core": "^1.119.0",
+        "@aws-cdk/lambda-layer-kubectl": "^1.119.0",
```

Comment:
Why do we need this dependency on lambdalayer?

Reply:
addressed

Reply:
Removed based on our conversation